docs/articles/expensify-classic/get-paid-back/expenses/Merge-Expenses.md

```diff
@@ -1,5 +1,62 @@
+2. You can only merge two expenses at a time.
+3. You can merge a cash expense with a credit card expense, or two cash expenses - but not two credit card expenses.
+4. In order to merge, both expenses will need to be in an Open or Unreported state.
+# How to merge expenses on the web app
+To merge two expenses from the Expenses page:
+1. Sign into your Expensify account.
+2. Navigate to the Expenses page on the left-hand navigation.
+3. Click the checkboxes next to the two expenses you wish to merge.
+4. Click **Merge**.
+5. You'll be able to choose which aspect of each of the two expenses you would like to be used on the resulting expense, such as the receipt image, card, merchant, category, and more.  
+
+To merge two expenses from the Reports page:
+1. Sign into your Expensify account.
+2. Navigate to the Reports page on the left-hand navigation.
+3. Click the Report that contains the expenses that you wish to merge.
+4. Click on the **Details** tab, then the Pencil icon.
+5. Select the two expenses that you wish to merge.
+6. You'll be able to choose which aspect of each of the two expenses you would like to be used on the resulting expense, such as the receipt image, card, merchant, category, and more.   
+# How to merge expenses on the Expensify mobile app
+On the mobile app, merging is prompted when you see the message "_Potential duplicate expense detected_". Simply tap **Resolve Now** to take a closer look, then hit **Merge Expense**, and you're done! 
+
+If the expenses exist on two different reports, you will be asked which report you'd like the newly created single expense to be reported onto.
+
+# FAQ
+
+## Can you merge expenses across different reports?
+
+You can not merge expenses across different reports. Expenses will only merge if they are on the same report. If you have expenses across different reports that you wish to merge, you’ll need to move both expenses onto the same report (and ensure they are in the Open status) in order to merge them.
+
+## Can you merge expenses across different accounts?
+
+You cannot merge expenses across two separate accounts. You will need to choose one submitter and transfer the expense information to that user's account in order to merge the expense.
+## Can you merge expenses with different currencies?
+
+Yes, you can merge expenses with different currencies. The conversion amount will be based on the daily exchange rate for the date of the transaction, as long as the converted rates are within +/- 5%. If the currencies are the same, then the amounts must be an exact match to merge.
+
+## Can Expensify automatically merge a cash expense with a credit card expense?
+
+Yes, Expensify can merge a cash expense with a credit card expense. A receipt will need to be SmartScanned via the app or forwarded to receipts@expensify.com in order to merge with a card expense. Note that the SmartScan must be fully completed and not stopped or edited, otherwise the two won’t merge. 
```

Comment:
```suggestion
Yes, Expensify can merge a cash expense with a credit card expense. A receipt will need to be SmartScanned via the app or forwarded to [receipts@expensify.com](mailto:receipts@expensify.com) in order to merge with a card expense. Note that the SmartScan must be fully completed and not stopped or edited, otherwise the two won’t merge. 
```